Resources/views/full/folder.html.twig

```diff
@@ -0,0 +1,45 @@
+{% extends "eZDemoBundle::pagelayout.html.twig" %}
+
+{% block content %}
+    <section class="content-view-full">
+        <div class="class-folder">
+            <div class="row">
+                <div class="col-md-8">
+                    <div class="attribute-header">
+                        <h1>{{  ez_render_field( content, 'name' ) }}</h1>
```

Comment:
nitpick: due this is our demobundle i will maybe use ```{{ ez_content_name (content) }}``` just to let the newcomers know about it.